server.js

```diff
@@ -1,33 +1,107 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import booksData from './data/books.json';
+
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
-// Start defining your routes here
+// Routes start here:
+// Home: path /
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send("Hello world, welcome to Vanessa's Bookish API! - See this API live at: https://vane-bookish-app.netlify.app/")
+});
+
+// Books: path /books - implemented pagination, each page will show 50 books
+// at a time (we have 500 books, so we have a max of 10 pages)
+// path example: /books?p=1
+app.get('/books', (req, res) => {
+  const postCount = booksData.length;
+  const perPage = 50;
+  // pageCount: counts amount of pages we end up with depending on our data's length
+  const pageCount = Math.ceil(postCount / perPage);
+
+  // page: determines value for the p variable in query - path: /books?p=3
+  let page = parseInt(req.query.p);
+  if(page < 1) page = 1;
```

Comment:
Would be nice if this caught if the user didn't write a `p` query at all as well :)